src/DotNetWorker.Grpc/GrpcWorker.cs

```diff
@@ -36,6 +36,7 @@ internal class GrpcWorker : IWorker
         private readonly IInputConversionFeatureProvider _inputConversionFeatureProvider;
         private readonly IMethodInfoLocator _methodInfoLocator;
         private readonly IOptions<GrpcWorkerStartupOptions> _startupOptions;
+        private readonly IOptions<WorkerOptions> _workerOptions;
```

Comment:
Does functions worker support reloading of config? If so, we should consider `IOptionsMonitor<>` here, so users can change this behavior while their app is running (without needing to re-deploy).

Reply:
noting we discussed offline - since the worker gets this flag from worker options then sends it as a capability to the host in the worker init request code path, this would require more of a design discussion on general dynamic refreshing of options/capabilities.